[2025-01-06T17:01:06.471+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T17:01:06.506+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.start_task manual__2025-01-06T17:01:03.433436+00:00 [queued]>
[2025-01-06T17:01:06.520+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.start_task manual__2025-01-06T17:01:03.433436+00:00 [queued]>
[2025-01-06T17:01:06.522+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T17:01:06.551+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): start_task> on 2025-01-06 17:01:03.433436+00:00
[2025-01-06T17:01:06.562+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=100) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T17:01:06.566+0000] {standard_task_runner.py:72} INFO - Started process 101 to run task
[2025-01-06T17:01:06.564+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'start_task', 'manual__2025-01-06T17:01:03.433436+00:00', '--job-id', '809', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpzcd58a6m']
[2025-01-06T17:01:06.567+0000] {standard_task_runner.py:105} INFO - Job 809: Subtask start_task
[2025-01-06T17:01:06.641+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.start_task manual__2025-01-06T17:01:03.433436+00:00 [running]> on host 97bbb29a7501
[2025-01-06T17:01:06.869+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='start_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T17:01:03.433436+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T17:01:03.433436+00:00'
[2025-01-06T17:01:06.873+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T17:01:06.982+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-06T17:01:06.984+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Here we start! "']
[2025-01-06T17:01:07.002+0000] {subprocess.py:99} INFO - Output:
[2025-01-06T17:01:07.006+0000] {subprocess.py:106} INFO - Here we start!
[2025-01-06T17:01:07.007+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-06T17:01:07.055+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T17:01:07.057+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=start_task, run_id=manual__2025-01-06T17:01:03.433436+00:00, execution_date=20250106T170103, start_date=20250106T170106, end_date=20250106T170107
[2025-01-06T17:01:07.109+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T17:01:07.159+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-06T17:01:07.164+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
